go.mod

```diff
@@ -323,6 +322,7 @@ require (
 	github.com/yashtewari/glob-intersection v0.1.0 // indirect
 	github.com/yusufpapurcu/wmi v1.2.4 // indirect
 	go.elastic.co/fastjson v1.1.0 // indirect
+	go.mozilla.org/pkcs7 v0.0.0-20210826202110-33d05740a352 // indirect
```

Comment:
Note that the dependency is still there (though now as `indirect` because unused in our codebase), that's because the version of `micromdm` we use still uses it. When we upgrade `micromdm` as part of the larger-scope https://github.com/fleetdm/fleet/issues/18057 story, this will be gone.